.github/ISSUE_TEMPLATE/hardware_support.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Hardware Support
+about: Suggest adding hardware support
+title: ''
+labels: Hardware Support
+assignees: ''
+
+---
+
+**Is this request related to a missing driver, SoC or board? Please describe.**
+Describe in details the hardware support being requested and why this support benefits Zephyr.
```

Comment:
"If this is a new board or SoC, please state whether you are willing to maintain the Zephyr support for it if it is included in the main tree"